code/addons/test/package.json

```diff
@@ -115,12 +115,14 @@
     "tree-kill": "^1.2.2",
     "ts-dedent": "^2.2.0",
     "typescript": "^5.3.2",
+    "vite": "^4.0.0",
     "vitest": "^2.1.3"
   },
   "peerDependencies": {
     "@vitest/browser": "^2.1.1",
     "@vitest/runner": "^2.1.1",
     "storybook": "workspace:^",
+    "vite": "^4.0.0 || ^5.0.0 || ^6.0.0",
```

Comment:
not a huge fan of this change but I don't see any other way to check which Vite version is installed in the user's project.

Reply:
You can either do that or check for the version in package.json

If you use the vite package, can you add it to the peerDependenciesMeta as optional?

Reply:
> You can either do that or check for the version in package.json

I'd rather not get into a mess of resolving packages manually, monorepos, symlinks, etc.

> If you use the vite package, can you add it to the peerDependenciesMeta as optional?

if it's not installed do you just suggest we assume it's Vite 5, not adding conditions? I'm also not sure when it would ever _not_ be there, given it's a peerDep of `@storybook/builder-vite`.

Reply:
We decided that the component tests addon will replace the interactions addon. That means Webpack users would be using it too. The addon has two paths:
- manager/preview entries for the addon panel
- vitest plugin/server channel for Vite based projects (optional)

That's why all vitest based dependencies must be declared optional

Reply:
ah of course. I'll take care of that.